src/core/resolve.js

```diff
@@ -79,10 +82,10 @@ export function relative(modulePath, sourceFile, settings) {
     function v1() {
       try {
         const path = resolver.resolveImport(modulePath, sourceFile, config)
-        if (path === undefined) return { found: false }
-        return { found: true, path }
+        if (path === undefined) return { found: false, path: null }
+        return { found: true, path: null }
```

Comment:
As i remember, it was due to [this](https://github.com/benmosher/eslint-plugin-import/blob/v1.8.1/resolvers/webpack/test/config.js#L30) failing test after merge of `master`.